README.md

```diff
@@ -35,7 +35,7 @@ Highlights:
 
 Writing zero-overhead, optimized WebAssembly is pretty tough to do. The syntax for `.wat` files is terse and difficult to work with directly. If you do not wish to use a systems language like C or Rust,
 then you're kind of out of luck. Your best bet (currently) is to write very plain C code, compile that to .wast and then optimize that result. Then you're ready to compile that into the final WebAssembly binary. This is an
-attempt to take C/Rust out of the equation and write 'as close to the metal' as possible without loosing readability.
+attempt to take C/Rust out of the equation and write 'as close to the metal' as possible without loozing readability.
```

Comment:
How about making the typo _better_?

Reply:
thats implying itll still be a typo when its made better, so do i just remove the extra "o" and keep the "z", otherwise it wont be a typo